src/tools/rbd_mirror/image_sync/ImageCopyRequest.cc

```diff
@@ -198,6 +200,11 @@ void ImageCopyRequest<I>::handle_object_copy(int r) {
       }
     }
 
+    if (!m_updating_sync_point &&
```

Comment:
I'd rather see the SafeTimer be utilized to periodically update the sync point every X seconds (new config option).